mlkem/indcpa.c

```diff
@@ -124,8 +124,10 @@ static void unpack_ciphertext(polyvec *b, poly *v,
   poly_decompress(v, c + MLKEM_POLYVECCOMPRESSEDBYTES);
 }
 
+#ifndef GEN_MATRIX_NBLOCKS
```

Comment:
Please namespace this. At least it should start with `MLKEM_`

Reply:
Ok